Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -85,27 +86,48 @@ namespace gdjs {
       runtimeGame: gdjs.RuntimeGame;
       gameId: string;
       connectionId?: string;
-    }) =>
-      `https://gd.games/auth?gameId=${gameId}${
+    }) => {
+      // Uncomment to test the case of a failing loading:
+      // return 'https://gd.games.wronglink';
+
+      return `https://gd.games/auth?gameId=${gameId}${
         connectionId ? `&connectionId=${connectionId}` : ''
       }${
         runtimeGame.isUsingGDevelopDevelopmentEnvironment() ? '&dev=true' : ''
       }`;
+    };
 
     /**
      * Helper returning the platform.
      */
     const getPlatform = (
       runtimeScene: RuntimeScene
     ): 'electron' | 'cordova' | 'web' => {
-      const electron = runtimeScene.getGame().getRenderer().getElectron();
+      const runtimeGame = runtimeScene.getGame();
+      const electron = runtimeGame.getRenderer().getElectron();
       if (electron) {
         return 'electron';
       }
       if (typeof cordova !== 'undefined') return 'cordova';
+
       return 'web';
     };
 
+    /**
+     * Check if, in some exceptional cases, we allow authentication
+     * to be done through a iframe.
+     * This is usually discouraged as the user can't verify that the authentication
+     * window is a genuine one. It's only to be used in trusted contexts.
+     */
+    const shouldWebAuthenticationUsingIframe = (runtimeScene: RuntimeScene) => {
```

Comment:
`shouldAuthenticationUseIframe` is more correct I think

Reply:
Fixed